charts/MapData.ts

```diff
@@ -426,6 +426,10 @@ export class MapData extends ChartTransform {
         return colorScheme !== undefined ? colorScheme : this.defaultColorScheme
     }
 
+    @computed get singleColorScale(): boolean {
+        return this.colorScheme?.singleColorScale
```

Comment:
Can `colorScheme` be undefined? If it can, we should change its type!

Reply:
Ah it cannot -- must've been a fragment of an earlier revision I had. Fixed and will push.